types/splitting/splitting-tests.ts

```diff
@@ -4,19 +4,20 @@ import "splitting/dist/splitting-cells.css";
 
 // Base splitting function
 
-Splitting();
-Splitting({});
-Splitting({
+Splitting(); // $ExpectType Result[]
+Splitting({}); // $ExpectType Result[]
+Splitting({ // $ExpectType Result[]
     target: ".target",
     by: "chars",
     key: "foo",
 });
 
+// $ExpectType Result[]
 Splitting({ key: null });
 
-Splitting({ target: document.createElement("span") });
-Splitting({ target: [document.createElement("span")] });
-Splitting({ target: document.querySelectorAll(".target") });
+Splitting({ target: document.createElement("span") }) // $ExpectType Result[]
```

Comment:
```suggestion
Splitting({ target: document.createElement("span") }); // $ExpectType Result[]
```

Minor formatting issue here, but the change from an object to an array looks right to me!